missilewars-plugin/src/main/java/de/butzlabben/missilewars/util/geometry/Geometry.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * This file is part of MissileWars (https://github.com/Butzlabben/missilewars).
+ * Copyright (c) 2018-2021 Daniel Nägele.
+ *
+ * MissileWars is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MissileWars is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MissileWars.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.butzlabben.missilewars.util.geometry;
+
+import org.bukkit.Location;
+import org.bukkit.World;
+
+public class Geometry {
+
+    /**
+     * This method checks if a location is closer to a target location
+     * than another location.
+     *
+     * @param targetLocation (Location) the start location for the distance measure
+     * @param closerLocation (Location) the closer location
+     * @param furtherAwayLocation (Location) the location that is further away
+     * @return true, if the statement is correct
+     */
+    public static boolean isCloser(Location targetLocation, Location closerLocation, Location furtherAwayLocation) {
+        if (targetLocation.distanceSquared(closerLocation) < targetLocation.distanceSquared(furtherAwayLocation)) return true;
+
+        return false;
```

Comment:
Use simplified version (IntelliJ also recommends it), sorry for not noticing earlier
```suggestion
        return targetLocation.distanceSquared(closerLocation) < targetLocation.distanceSquared(furtherAwayLocation);
```